src/observer.ts

```diff
@@ -52,6 +52,19 @@ export function observer<P extends object, TRef = {}>(
         memoComponent = memo(wrappedComponent)
     }
 
+    copyStaticProperties(baseComponent as any, memoComponent as any)
```

Comment:
Why `any`? Wouldn't be easier to just inline that code instead of having a function that's not really reused? Or even inline a whole function if you want to name that operation, but you could use a closure that way.

Reply:
The goal of the function is not reuse, it is to make the narrative clearer for future readers. `observer`'s goal is to wrap the function we're passing. As an essential detail we must hoist statics, but how that is done is really an implementation detail. Keeping functions small makes it much easier for future readers to grab what is globally happening, what is happening in detail, and what statements go together and provide some cohesive functionality.